tests/client/disco/containers/TestInstallButton.js

```diff
@@ -37,6 +49,7 @@ describe('<InstallButton />', () => {
   });
 
   it('should reflect UNINSTALLED status', () => {
+    stubAddonManager();
```

Comment:
I wonder if the test could be organised so the tests that have the common `stubAddonManager()` can have it run in a `beforeEach()`. It's not the end of the world but might save on some repetition.